cliquet/views/hello.py

```diff
@@ -13,6 +13,7 @@ def get_hello(request):
     data = dict(
         hello=project_name,
         version=settings['project_version'],
+        protocol_version=API_VERSION,
```

Comment:
I would be in favor of having two constants, it will clarify to the reader that they can differ. That's a nitpick though :)